modules/codegen/src/smithy4s/codegen/SmithyToIR.scala

```diff
@@ -273,7 +273,8 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
                     value.getName().asScala,
                     value.getValue,
                     index
-                  )
+                  ),
+                  hints = Nil
```

Comment:
AFAIK 1.x enums don't allow hints on values.

Reply:
That is correct